engine/table/src/main/java/io/deephaven/engine/table/impl/sort/findruns/ByteFindRunsKernel.java

```diff
@@ -94,32 +117,66 @@ private static void findRuns(ByteChunk sortedValues, int offset, int length, Wri
         }
     }
 
-    private static boolean neq(byte last, byte next) {
-        // region neq
-        return next != last;
-        // endregion neq
+    public static int compactRuns(
+            @NotNull final WritableByteChunk<? extends Any> sortedValues,
+            @NotNull final IntChunk<ChunkPositions> offsetsIn) {
+        final int numRuns = offsetsIn.size();
+        if (numRuns == 0) {
+            return -1;
+        }
+        byte last = sortedValues.get(0);
```

Comment:
Should this be offsetsIn.get(0) and should we set sortedValues[0] to that value?

Reply:
Hmm. I sort of assumed offsets would have to start at 0. I suppose I could change it. Fixed. I'll write a unit test.

Reply:
Maybe not about that unit test.